### PR TITLE
Hawkstrider part1

### PR DESCRIPTION
**Pull request by LeeDarren**
Attempting a smaller pull request, hopefully under the 100 line limit.

### CODE REVIEW

**Comment by LeeDarren** [comment]
Please review this, @casey-collab

File: english/hawkstrider/startr/startr.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+The hawkstrider pauses for a moment then says:
+
+"Reaarrr. Urrrrr."
+
+A. [How ya doin fam?](../error/error.md)
+
+B. [Doral ana-diel?](../tier1/tier2/tier2.md)
```
Comment (casey-collab) [inline english/hawkstrider/startr/startr.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/hawkstrider/startr/startr.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+C. [Rearrrrr](../tier1/tier1.md)
```
Comment (casey-collab) [inline english/hawkstrider/startr/startr.md]: This link is also invalid.

File: english/hawkstrider/startu/startu.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+The hawkstrider pauses for a moment then says:
+
+"Urrrrrrrrr."
+
+A. [How ya doin fam?](../error/error.md)
+
+B. [Doral ana-diel?](../tier1/tier2/tier2.md)
```
Comment (casey-collab) [inline english/hawkstrider/startu/startu.md]: This link is also invalid.

File: english/hawkstrider/startu/startu.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+C. [Rearrrrr](../tier1/tier1.md)
```
Comment (casey-collab) [inline english/hawkstrider/startu/startu.md]: This link is also invalid.

File: english/hawkstrider/rude/rude.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+The hawkstrider leaps back, evidently startled.  It glares angrily at you for a moment,
```
Comment (casey-collab) [inline english/hawkstrider/rude/rude.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/hawkstrider/rude/rude.md (hunk 5 above)
Comment (LeeDarren) [inline english/hawkstrider/rude/rude.md]: It should be reachable with the full push. Please review this, @casey-collab

**Comment by LeeDarren** [comment]
Please review this again, the code is reachable with the full push. @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, pull requests adding more than 100 lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to add 100 lines or fewer.

**Comment by LeeDarren** [comment]
I'm closing this request.  My full push has over 100 lines so the bot won't merge it.  If it break it into the earlier commits, there are broken links(due to missing files from later commits) and the bot won't let me merge it.  If I push the commits with the fixes, then my push is over the 100 line limit, so the bot won't merge it.  The only way I can see to fix it is to intentionally break my working files to get them to lead to the working files and push commits that are intentionally broken.  I will just hope someone merges my first pull request.

**Pull request closed without merging**